packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -178,6 +178,7 @@ export default function PostLockedModal() {
 			shouldCloseOnClickOutside={ false }
 			shouldCloseOnEsc={ false }
 			isDismissible={ false }
+			className="editor-post-locked-modal"
```

Comment:
Can you add a comment explaining that this is here because it used by plugins.

Also worth nothing that classnames are not public APIs officially but we can keep them around if it doesn't cost core anything in terms of complexity/performance.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/64257/commits/14e9eb65afad4034fff3be409a69d2e94eabcf1b. 

@youknowriad What is the recommend way to extend or override this component if not using this class? At the moment, I can't see a way.

Reply:
Fixed in https://github.com/WordPress/gutenberg/commit/14e9eb65afad4034fff3be409a69d2e94eabcf1b.

@youknowriad What is the recommend way to extend or override this component if not using this class? At the moment, I can't see a way.

Reply:
There is no official way at the moment. In general it might be advised to raise an issue to discuss the use-case (and related use-cases) to see if we can come up with a good API.